services/enrichment/manager/src/events/onFileIngested.js

```diff
@@ -26,6 +26,9 @@ export const handler = async (event, context, callback) => {
   // Get Account ID from lambda function arn in the context
   const accountId = context.invokedFunctionArn.split(':')[4];
 
+  const sns = new AWS.SNS();
+  const snsEndpoint = `arn:aws:sns:${REGION}:${accountId}:${STAGE}-onEnrichRecordRequested`;
```

Comment:
I would suggest skipping `Record` and have it `onEnrichmentRequested`. A bit simpler and also leaves it fine in case you decide to bulk/batch enrich.

Reply:
`onEnrichmentRequested` already exists